runtime/mangata-kusama/src/lib.rs

```diff
@@ -1655,13 +1684,13 @@ construct_runtime!(
 		Sudo: pallet_sudo_mangata::{Pallet, Call, Config<T>, Storage, Event<T>} = 49,
 		SudoOrigin: pallet_sudo_origin::{Pallet, Call, Event<T>} = 50,
 		Council: pallet_collective_mangata::<Instance1>::{Pallet, Call, Storage, Origin<T>, Event<T>, Config<T>} = 51,
+		Identity: pallet_identity::{Pallet, Call, Storage, Event<T>} = 56,
```

Comment:
is there some convention which number to use ?

Reply:
i guess you can consider updating pallets Id, the only thing you need on top of that seems to be incrementation of transaction_version of runtime version struct, but it has been done already as thats something we do with every release :

https://github.com/mangata-finance/substrate/blob/8d5b532ef66f0f06be2e9b23c0f8ecf335440075/primitives/version/src/lib.rs#L201-L202

https://github.com/mangata-finance/mangata-node/blob/56c044d97da917b005ec7dfa696f657c3917e557/runtime/mangata-rococo/src/lib.rs#L181